From: John Hubbard <jhubbard@nvidia.com>
Subject: selftests/mm: fix unused variable warning in migration.c
Date: Thu, 1 Jun 2023 18:33:49 -0700

Declare the variable as volatile, in order to avoid the clang compiler
warning.

Link: https://lkml.kernel.org/r/20230602013358.900637-4-jhubbard@nvidia.com
Signed-off-by: John Hubbard <jhubbard@nvidia.com>
Cc: Mel Gorman <mgorman@techsingularity.net>
Cc: Johannes Weiner <hannes@cmpxchg.org>
Cc: Mike Kravetz <mike.kravetz@oracle.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/testing/selftests/mm/migration.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/tools/testing/selftests/mm/migration.c~selftests-mm-fix-unused-variable-warning-in-migrationc
+++ a/tools/testing/selftests/mm/migration.c
@@ -95,7 +95,7 @@ int migrate(uint64_t *ptr, int n1, int n
 
 void *access_mem(void *ptr)
 {
-	uint64_t y = 0;
+	volatile uint64_t y = 0;
 	volatile uint64_t *x = ptr;
 
 	while (1) {
_
